### ISSUE TITLE
"warnings" use sys.stderr.write() instead of warnings.warn()

### ISSUE THREAD

**Issue by kforeman** [yhat/ggplot#411]
By using `sys.stderr.write()` it leaves users unable to capture and manage warnings in ggplot. E.g. if the width and height of a plot aren't specified [these lines in savefig()](https://github.com/yhat/ggplot/blob/fa84a244ee2e89763723b06b00510481d0001409/ggplot/utils/ggutils.py#L141-L143) write a warning to stderr:

```python
if issue_size:
    msg = "Saving {0} x {1} {2} image.\n".format(from_inch[units](width), from_inch[units](height), units)
    sys.stderr.write(msg)
```

If Python's built in [warnings module](https://docs.python.org/2/library/warnings.html) (`warnings.warn()`) were used instead, then users could filter out irrelevant warnings or do something in response to specific warnings.

I can submit a PR with these changes but first want to make sure there's no philosophical reason for using stderr instead.

**Comment by JanSchulz** [comment]
Please do, thats actually somewhere on the ToDo list...

**Comment by glamp** [comment]
fixed in 0.9.3
